beacon-chain/sync/service_test.go

```diff
@@ -122,9 +123,6 @@ func TestProcessBlock(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Sync service broadcasts the block and forwards the block to to the local chain.
-	testutil.AssertLogsContain(t, hook, fmt.Sprintf("Broadcasting block hash to peers: %x", h))
```

Comment:
Is there a reason why you didn't replace this with `testutil.AssertLogsContain(t, hook, fmt.Sprintf("Relaying processed block hash to peers: %x", h))`?